modules/redshift-resources/variables.tf

```diff
@@ -0,0 +1,37 @@
+# Variables for Redshift Lambda UDF Module
+
+variable "deployment_name" {
```

Comment:
Is it worth adding the new variables to the optional section of `terraform.tfvars.example`?